pandas/tests/plotting/frame/test_frame.py

```diff
@@ -840,14 +840,26 @@ def test_plot_scatter_shape(self):
         axes = df.plot(x="x", y="y", kind="scatter", subplots=True)
         _check_axes_shape(axes, axes_num=1, layout=(1, 1))
 
-    def test_raise_error_on_datetime_time_data(self):
-        # GH 8113, datetime.time type is not supported by matplotlib in scatter
+    def test_scatter_on_datetime_time_data(self):
+        # datetime.time type is now supported in scatter, since a converter
+        # is implemented in ScatterPlot
         df = DataFrame(np.random.default_rng(2).standard_normal(10), columns=["a"])
         df["dtime"] = date_range(start="2014-01-01", freq="h", periods=10).time
-        msg = "must be a string or a (real )?number, not 'datetime.time'"
-
-        with pytest.raises(TypeError, match=msg):
-            df.plot(kind="scatter", x="dtime", y="a")
+        df.plot(kind="scatter", x="dtime", y="a")
+
+    def test_scatter_line_xticks(self):
+        # GH#61005
+        datetime_list = [datetime(year=2025, month=1, day=1, hour=n) for n in range(3)]
+        df = DataFrame(columns=["datetime", "y"])
+        for i, n in enumerate(datetime_list):
+            df.loc[len(df)] = [n, i]
```

Comment:
Could you create this in 1 call? i.e. pass a `dict` to `DataFrame`

Reply:
I created it by passing a list